css/style.css

```diff
@@ -0,0 +1,73 @@
+body{
+	padding: 40px 20px;
+		font-family: sans-serif;
```

Comment:
:hammer: Your code looks great. Just beware about proper indentation for best practices :)